Doesn't work with Godot 3.0

Probably needs the same treatment as the open simplex got.

```
In file included from modules/fastnoise/lib/FastNoise.cpp:29:0:
modules/fastnoise/lib/FastNoise.h:133:37: error: 'nullptr' was not declared in this scope
  FastNoise* m_cellularNoiseLookup = nullptr;
                                     ^
[ 14%] modules/fastnoise/lib/FastNoise.cpp: In member function 'float fastnoise::FastNoise::GetNoise(float, float, float)':
modules/fastnoise/lib/FastNoise.cpp:254:18: error: 'fastnoise::FastNoise::NoiseType' is not a class or namespace
  case FastNoise::NoiseType::Value:
```

This error repeats for other NoiseType's aswell

--- a/SCsub
+++ b/SCsub
@@ -1,4 +1,9 @@
 Import('env')
 
-env.add_source_files(env.modules_sources,"*.cpp")
-env.add_source_files(env.modules_sources,"lib/*.cpp")
+module_env = env.Clone()
+
+module_env.add_source_files(env.modules_sources,"*.cpp")
+module_env.add_source_files(env.modules_sources,"lib/*.cpp")
+
+# FastNoise uses a few C++11 features
+module_env.Append(CPPFLAGS=["-std=c++11"])

--- a/config.py
+++ b/config.py
@@ -5,5 +5,4 @@ def can_build(platform):
   
   
 def configure(env):
-    # FastNoise uses a few C++11 features
-    env.Append(SCONS_CXX_STANDARD="c++11")
+    pass

